source/CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ if (BUILD_CPP_IF)
   if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 4.9)
     set (LIB_DEEPMD_NATIVE	"deepmd_native_md")
     set (LIB_DEEPMD_IPI		"deepmd_ipi")
+    set (LIB_DEEPMD_GROMACS "deepmd_gromacs")
   else ()
     message (STATUS "Your gcc/g++ version is ${CMAKE_CXX_COMPILER_VERSION}, so native MD and ipi are disabled. To enable them, use gcc/g++ >= 4.9.")
```

Comment:
This message should be updated.

Reply:
wait, I think the new json.hpp supports GCC 4.8?

Reply:
Sure.